components/customgpt/actions/send-message/send-message.mjs

```diff
@@ -0,0 +1,114 @@
+import { getFileStreamAndMetadata } from "@pipedream/platform";
+import FormData from "form-data";
+import {
+  CHATBOT_MODEL_OPTIONS,
+  RESPONSE_SOURCE_OPTIONS,
+} from "../../common/constants.mjs";
+import customgpt from "../../customgpt.app.mjs";
+
+export default {
+  key: "customgpt-send-message",
+  name: "Send Message",
+  description: "Sends a message to an existing conversation within a project. [See the documentation](https://docs.customgpt.ai/reference/post_api-v1-projects-projectid-conversations-sessionid-messages)",
+  version: "0.0.1",
+  annotations: {
+    destructiveHint: false,
+    openWorldHint: true,
+    readOnlyHint: false,
+  },
+  type: "action",
+  props: {
+    customgpt,
+    projectId: {
+      propDefinition: [
+        customgpt,
+        "projectId",
+      ],
+    },
+    conversationId: {
+      propDefinition: [
+        customgpt,
+        "conversationId",
+        ({ projectId }) => ({
+          projectId,
+          fieldId: "session_id",
+        }),
+      ],
+    },
+    externalId: {
+      type: "string",
+      label: "External ID",
+      description: "The external ID of the message",
+      optional: true,
+    },
+    prompt: {
+      type: "string",
+      label: "Prompt",
+      description: "Prompt to send to OpenAI",
+      optional: true,
```

Comment:
Prompt is required. See [API Doc here](https://docs.customgpt.ai/reference/post_api-v1-projects-projectid-conversations-sessionid-messages)
```suggestion
```